### ISSUE TITLE
RetroArch crashes when using “refresh playlist” from the sidebar quick access menu in Ozone

### ISSUE THREAD

**Issue by trotsky40** [libretro/RetroArch#16316]
## Description

RetroArch crashes when using “refresh playlist” from the sidebar quick access menu in the latest nightly. It does not crash when refreshing using the settings playlist menu. 

Similar to this issue but was not resolved with the fix: https://github.com/libretro/RetroArch/issues/16295

### Steps to reproduce the bug

1. Open sidebar quick access menu for a playlist 
2. Select “refresh”
3. RetroArch appears to complete the action and then will crash back to the desktop

### Version/Commit
You can find this information under Information/System Information

- RetroArch: Observed on latest nightly (git version 75c0ede) for both Windows and Linux (Steam Deck via app image)

### Environment information

- Compiler: from build bot

**Comment by sonninnos** [comment]
Not crashing here after doing a manual scan so that refresh is available, so please provide more details. Tried having one playlist only and more.

**Comment by trotsky40** [comment]
I am on Windows 10 using the Ozone menu. I have about 60 playlists that were all done with manual scans. The refresh from the quick access menu does intermittently work. Playing around with it, the crash is not on the refresh itself but when exiting the quick access menu after the refresh. The log doesn't provide anything useful but including it for completeness. [retroarch__2024_03_02__12_37_14.log](https://github.com/libretro/RetroArch/files/14470712/retroarch__2024_03_02__12_37_14.log)

**Comment by sonninnos** [comment]
Windows 10 here too and of course Ozone since no other menu has the sidebar. Nothing happens when I return from the playlist manager page after refresh is done.

**Comment by trotsky40** [comment]
Recorded on my steam deck (I see the same behavior) in case I am describing it incorrectly. 

https://drive.google.com/file/d/1J4kPZL-oW_14ld6OhxDmrXxnDyj08Wnh/view?usp=drivesdk

**Comment by bslenul** [comment]
I *think* the crash has nothing to do with the "Refresh" setting but is due to the menu scrolling somehow, let me explain: if you you exit the "Manage" menu while it is scrolled down, RA will crash, if you exit the menu while it's at its default position then no issue:

https://github.com/libretro/RetroArch/assets/33353403/247f238b-5dc7-4a72-af90-1362df62ec81

^ here for example I scroll down then up again to "restore" the default menu position and no issue, then I scroll down and exit the menu and RA crashes.

So basically to reproduce you'll have to get the height small enough to be able to scroll.

Here's a crash log: [crash-240303-020725.log](https://github.com/libretro/RetroArch/files/14471325/crash-240303-020725.log)

**Comment by sonninnos** [comment]
Ahaa, thanks, confirmed & on it.
